### PR TITLE
Add install instructions for vim 8+

### PR DESCRIPTION
**Pull request by solderjs**
Add simple instructions for using syntastic with native vim 8 packages.

### CODE REVIEW

**Comment by lcd047** [comment]
What problem does your patch solve?  According to the README file:

> Starting with Vim version 7.4.1486 you can also load syntastic using the standard mechanism of packages, without the help of third-party plugin managers (see `:help packages` in Vim for details).

This seems to be clear enough, nobody has ever expressed any confusion about it.

**Comment by solderjs** [comment]
Now that macOS ships with Vim that supports plugins, it makes to me to make the standard way more prominent (and perhaps make the old way less prominent).

I never saw that notice before (buried in lots of text) - and it's not clear to me.

Just trying to make it as clear how to install this the new standard way as it is clear how to install the old way with pathogen.

**Comment by lcd047** [comment]
There isn't anything special about syntastic, it can be installed the same way as any other Vim plugin.  Syntastic has now been obsoleted by [ALE][ALE] (which can do asynchronous checking and interface with language servers), so it's only real use these days is for situations where you need to run it under Vim 7.  Hence the reference to Pathogen, since Vim 7 added a package manager only in the late stages.

TL;DR: there is little point in grooming a dead horse.

[ALE]: https://github.com/dense-analysis/ale

**Comment by solderjs** [comment]
I've never heard of Ale. If that's the replacement... should not that go in the README as well?
